multimedia/minisatip/files/minisatip.init

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh /etc/rc.common
+
+USE_PROCD=1
+
+start_service() {
+  procd_open_instance [minisatip]
+  procd_set_param command /usr/bin/minisatip -f -R /usr/share/minisatip/html
+
+  # respawn automatically if something died, be careful if you have an alternative process supervisor
+  # if process dies sooner than respawn_threshold, it is considered crashed and after 5 retries the service is stopped
+  procd_set_param respawn ${respawn_threshold:-3600} ${respawn_timeout:-5} ${respawn_retry:-5}
+
+  procd_set_param stdout 1 # forward stdout of the command to logd
+  procd_set_param stderr 1 # same for stderr
+  procd_set_param user root
+  procd_set_param pidfile /var/run/minisatip.pid # write a pid file on instance start and remote it on stop
```

Comment:
Maybe the user and pidfile param are redundant here.  `root` is the default.  What's the pidfile file for?  btw. a typo in the comment `remote -> remove`

Reply:
Please use tab for indentation